purchases/src/integrationTest/AndroidManifest.xml

```diff
@@ -1,13 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:tools="http://schemas.android.com/tools"
+        xmlns:android="http://schemas.android.com/apk/res/android">
 
     <application
             android:allowBackup="true"
             android:icon="@mipmap/ic_launcher"
             android:label="@string/app_name"
             android:roundIcon="@mipmap/ic_launcher_round"
             android:supportsRtl="true"
-            android:theme="@style/Theme.Purchasesandroid">
+            android:debuggable="true"
```

Comment:
We shouldn't do this for a normal app... but since this is integration tests I think it's fine.

Reply:
yeah, for testing app it's totally fine